components/Subscription/index.jsx

```diff
@@ -0,0 +1,29 @@
+import clsx from 'clsx'
+import styles from './styles.module.scss'
+
+const Subscription = ({ title, description }) => {
+    return (
+        <div className={styles.container}>
+            <div className={clsx(styles.title, 't-h3')}>{title}</div>
+            <div className={styles.row}>
+                <div className={clsx(styles.description, 't-button')}>
+                    {description}
+                </div>
+                <form className={styles.form}>
+                    <label>
+                        <input
+                            className={styles.address}
+                            type='email'
+                            placeholder='Your email address'
+                        />
+                    </label>
+                    <button className={clsx(styles.button, 't-body')}>
```

Comment:
Are you seeing these errors?
![image](https://user-images.githubusercontent.com/15239884/116726070-a21e9080-a997-11eb-91ae-8dd760422c94.png)